src/player.cpp

```diff
@@ -11083,25 +11143,25 @@ void player::forced_dismount()
     } else {
         add_msg( m_debug, "Forced_dismount could not find a square to deposit player" );
     }
-    if( g->u.get_grab_type() != OBJECT_NONE ) {
-        add_msg( m_warning, _( "You let go of the grabbed object." ) );
-        g->u.grab( OBJECT_NONE );
+    if( !is_player() ) {
```

Comment:
I assume you meant this
```suggestion
    if( is_player() ) {
```
Because calling `game::update_map` doesn't make sense for NPCs, as well as the grabbing code etc.